docs/open-source-project/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,631 @@
+    - A crash has been fixed when relayout is done by adding/removing child.
+    - Null handle of Renderer return error has been fixed.
+    - Crash of WatchApplication has been fixed.
+    - Transition, DynamicResource and Style related issues of NUI XAML have been fixed.
+
+
+## UI framework
+
+### New and changed features
+
+- EFL
+  - Version 1.21 is upgraded to 1.22.
+  - Meson build system support has been added (beta).
+  - Lottie animation viewer has been added.
+  - Evas map quality has been improved.
+- Atk
+  - Version 2.28.1 is upgraded to 2.30.0.
+- At-spi2-core
+  - Version 2.26.1 is upgraded to 2.31.1.
+- At-spi2-atk
+  - Version 2.26.1 is upgraded to 2.30.0.
+- Fontconfig
+  - Version 2.13.0 is upgraded to 2.13.1.
+- Fribidi
+  - Version 1.0.2 is upgraded to 2.0.5.
+- Harfbuzz
+  - Version 1.8.1 is upgraded to 2.4.0.
+- rlottie
+  - Lottie file support has been added.
+
+### Known Issues
+
+- Accessibility toolkit packages support meson build system only.
+  - Atk, At-spi2-core, and At-spi2-atk
+- Elementary uses legacy focus policy rather than the recent focus manager for compatibility issue.
+
+
+## Interaction Framework
+
+### New and changed features
+
+- Multi-assistant framework
+  - Tizen now supports multi-assistant framework that supports multiple assistants at the same time.
+- Speech recognition engine
+  - Added an embedded speech recognition engine as an STT (Speech To Text) engine. For supporting an environment without internet, Tizen developers can now optionally use an embedded based speech recognition engine in their application.
+- Autofill framework
+  - Added Autofill API set to allow user to fill out commonly entered information.
+- Input service framework
+  - Added IME API set getting key code and visibility state of candidate.
+
+
+## Multimedia framework
+
+### New and changed features
+
+- MediaTool
+  - API set to support audio channel mask has been added.
+- Media Player
+  - Audio Offloading support has been added.
+  - Some display type has been removed.
+  - API set for progressive download has been removed.
+  - player_get_streaming_download_progress pre-condition has been extended.
+  - The range of parameter values of buffering related function has been changed and the value of -1 has been allowed.
+  - Support for controlling the audio pitch has been added.
+- Media Controller
+  - Management of content season, episode, and resolution metadata has been added.
+  - Support for getting and counting playlists has been updated.
+  - Support for counting search conditions has been added.
+- Image Util
+  - image_util_calculate_buffer_size() has been deprecated.
+  - Sync transform API set (convert, crop, resize, and rotate) has been added.
+- Thumbnail Util
+  - Extract thumbnails even if requested size exceeds the source size.
+- Camera
+  - Change buffer sharing mechanism to improve buffer protection.
+  - Support user allocated buffer for zero copy preview.
```

Comment:
To maintain consistency, suggest rephrasing:

Support for user allocated buffer for zero copy preview has been added.

Please verify

Reply:
will accept